docker-compose example/docker-compose.example.yml

```diff
@@ -0,0 +1,110 @@
+      - apparmor:unconfined
+    environment:
+      PHOTOVIEW_DATABASE_DRIVER: ${PHOTOVIEW_DATABASE_DRIVER}
+      PHOTOVIEW_MYSQL_URL: "${MARIADB_USER}:${MARIADB_PASSWORD}@tcp(photoview-mariadb)/${MARIADB_DATABASE}"
+      PHOTOVIEW_LISTEN_IP: "photoview"
+      PHOTOVIEW_LISTEN_PORT: 80
+      PHOTOVIEW_MEDIA_CACHE: "/app/cache"
+      ## Optional: If you are using Samba/CIFS-Share and experience problems with "directory not found"
+      ## Enable the following Godebug
+      # - GODEBUG=asyncpreemptoff=1
+      ## Optional: To enable map related features, you need to create a mapbox token.
+      ## A token can be generated for free here https://account.mapbox.com/access-tokens/
+      ## It's a good idea to limit the scope of the token to your own domain, to prevent others from using it.
+      MAPBOX_TOKEN: ${MAPBOX_TOKEN}
+    ## Share hardware devices with FFmpeg (optional):
+    # devices:
+      ## Uncomment next devices mappings if they are available in your host system
+      ## Intel QSV
+      # - "/dev/dri:/dev/dri"
+      ## Nvidia CUDA
+      # - "/dev/nvidia0:/dev/nvidia0"
+      # - "/dev/nvidiactl:/dev/nvidiactl"
+      # - "/dev/nvidia-modeset:/dev/nvidia-modeset"
+      # - "/dev/nvidia-nvswitchctl:/dev/nvidia-nvswitchctl"
+      # - "/dev/nvidia-uvm:/dev/nvidia-uvm"
+      # - "/dev/nvidia-uvm-tools:/dev/nvidia-uvm-tools"
+      ## Video4Linux Video Encode Device (h264_v4l2m2m)
+      # - "/dev/video11:/dev/video11"
+    volumes:
+      ## Example:
+      ## - "/host/folder:/container/folder"
+      - "/etc/localtime:/etc/localtime:ro" ## use timezone from host
+      - "/etc/timezone:/etc/timezone:ro"   ## use timezone from host
+      - "${HOST_PHOTOVIW_LOCATION}/storage:/app/cache"
+      ## Change This in the .env file: to the directory where your photos are located on your server.
+      ## If the photos are located at `/home/user/photos`, then change this value
+      ## to the following: `/home/user/photos:/photos:ro`.
+      ## You can mount multiple paths, if your photos are spread across multiple directories.
+      ## The same path, as the container path set here, you'll need to provide on the Photoview's init page (the one between the ':' chars).
+      ## If you mounted several folders, provide the path to the parent one on the init page.
+      - "${HOST_PHOTOVIW_MEDIA_ROOT}:/photos:ro"
+      ## *Additional* media folders can be mounted like this (set variable in .env file):
+      # - "${HOST_PHOTOVIW_MEDIA_FAMILY}:/photos/Family:ro"
+
+  db:
+    image: mariadb:lts
+    container_name: photoview-mariadb
+    restart: unless-stopped
+    stop_grace_period: 5s
+    ## Optimized MariaDB startup command for better performance and compatibility
+    command: mariadbd --innodb-buffer-pool-size=512M --transaction-isolation=READ-COMMITTED --character-set-server=utf8mb4 --collation-server=utf8mb4_unicode_ci --max-connections=512 --innodb-rollback-on-timeout=OFF --innodb-lock-wait-timeout=120
```

Comment:
@kkovaletp Can these flags be added to the compose file when
- using mariadb:10.5 before
- and using without any flags before
without breaking the existing database?
Thanks!

Reply:
I guess, you're asking about the command flags in line 67. From my understanding, yes, but to make sure double-check the MariaDB docs. Also, I'd suggest doing a full DB backup and then setting up a new Photoview instance with my config, mapping to the new DB volume folder, running the DB, and restoring the backup. In this way, you ensure there is no data loss and corruption